indico/modules/events/templates/display/conference/base.html

```diff
@@ -5,19 +5,23 @@
 
 {%- macro _format_event_date(event) -%}
     {% set start_dt = event.start_dt_display.astimezone(event.display_tzinfo) %}
+    {% set start_dt_iso = event.start_dt_display.isoformat() %}
     {% set end_dt = event.end_dt_display.astimezone(event.display_tzinfo) %}
+    {% set end_dt_iso = event.end_dt_display.isoformat() %}
     {% if start_dt.date() == end_dt.date() %}
-        {{ start_dt | format_date('long') }}
+       <span itemprop="startDate" content="{{ start_dt_iso }}"> {{ start_dt | format_date('long') }}</span>
```

Comment:
there shouldn't be a space after the `>`

Reply:
Fixed